deploy/olm-catalog/opendatahub/1.2.0/manifests/opendatahub-operator.v1.2.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,357 @@
+
+    Open Data Hub operator allows users to install and manage Open Data Hub and Kubeflow components. Users can mix and match tools from each project to fulfill the needs of their use case. Each of the
+    projects share some components, but can be mostly seen as an extension of each other to provide a complete solution for both novice and skilled enterprise users.
+
+    ### Components
+    * JupyterHub v0.3.6 - Open source multi-user JupyterLab notebook platform w/ GPU support and **NEW** High Availability.
+    * Trino v355 - Distributed analytics SQL database that supports multiple data sources
+    * Hue v4.8.0 - A service that provides data exploration on Hive and S3 buckets.
+    * Spark Thrift Server - A service that allows JDBC clients run Spark SQL queries.
+    * Open Data Hub Dashboard v1.0  - A web dashboard that displays installed Open Data Hub components with easy access to component UIs and documentation
+    * Elyra v3.8.0 - JupyterLab notebooks with support for AI workflows
+    * Ceph Nano v0.7 - Minimal Object Storage provided by Ceph for development use
+    * Prometheus - Monitoring and alerting tool
+    * Grafana - Data visualization and monitoring
+    * Airflow v1.10.11 - Workflow management
+    * Seldon - Open source platform for deploying machine learning models
+    * Argo v2.12.5 - Container-native Workflow Engine
+    * Apache Superset v1.4.1 - **NEW** Open source application for data exploration and visualization
+    * Apache Kafka v2.8.0 - The open source stream processing platform
+    * OpenShift Pipelines - Cloud-native CI/CD on OpenShift
+    To install one or multiple of these components use the default KfDef provided with the operator.
+    NOTE: Components without versions are defaulting to the current operator version available in the OperatorHub. These version may change as the components update their operator releases in OperatorHub.
+
+    ### Kubeflow Components
+    ODH 1.2.0 supports Kubeflow v1.4.0 and some components such as KF Serving and KFP on Tekton from master branch. To install Kubeflow v1.4.0 components please use this example [KfDef](https://raw.githubusercontent.com/opendatahub-io/manifests/v1.4.0-rc.2-openshift/openshift/kfdef/kfctl_openshift_v1.4.0_odh.yaml).
+
+    NOTE: The Open Data Hub distribution of the Kubeflow 1.4 stack  currently runs on OpenShift versions of 4.8 (or lower). For latest updates, please visit [link](https://github.com/opendatahub-io)
+
+    ### OpenShift 4.9
+
+    For installing Open Data Hub on OpenShift 4.9, please use the provided sample [KfDef](https://github.com/opendatahub-io/odh-manifests/blob/master/kfdef/kfctl_openshift4.9_odh.yaml) that only includes JupyterHub. We are currently updating all other components to work on OpenShift 4.9 for the latest update please visit [ODH-522](https://issues.redhat.com/browse/ODH-522)
```

Comment:
Should we update this to note that all components except `Spark` are working on `4.9`

Reply:
Good point. I add a note